### PR TITLE
change NUM_LDAS_ENSEMBLE in one place

### PR DESCRIPTION
**Pull request by weiyuan-jiang**
It is more convenient and consistent to change NUM_LDAS_ENSEMBLE in just one place while debugging or testing

### CODE REVIEW

File: src/Applications/LDAS_App/lenkf.j.template [hunk 1]
```diff
@@ -75,6 +74,7 @@ endif
 
 if (! -e $SCRDIR            ) mkdir -p $SCRDIR
 
+set NENS = `grep NUM_LDAS_ENSEMBLE   $HOMDIR/LDAS.rc | cut -d':' -f2`
```
Comment (gmao-rreichle) [inline src/Applications/LDAS_App/lenkf.j.template:77]: Should there be a colon after "grep   NUM_LDAS_ENSEMBLE"?  There is a colon after the grep for END_DATE and NUM_SGMT.

Comment (weiyuan-jiang) [reply src/Applications/LDAS_App/lenkf.j.template:77]: It is not necessary. The grep gets the whole line and cut by ":".  But we can add it if we want the syntaxs to be consistent.

**Review by biljanaorescanin — APPROVED** [review]
This was during GEOSldas tagup meeting

**Comment by gmao-rreichle** [comment]
When this is merged, please add something like the following "squashed" commit statement:

"Determine NUM_LDAS_ENSEMBLE entry in lenkf.j from LDAS.rc, replacing the independent, ldas_setup-generated specification in lenkf.j"

**Comment by weiyuan-jiang** [comment]
will be addressed by #316

**Pull request closed without merging**